### PR TITLE
resouces for bash

### PR DESCRIPTION
**Pull request by gutierri**
add new section with articles and a book about bash shell scripting ( #39 )

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -214,6 +218,13 @@ Organized by the Linux Foundation, it's a great free course to learn basics abou
 
 If you used Windows and decided to switch to Linux, this course should help you to find their alternatives. It's pretty short and you can finish it in a day. - [@aleksandar-todorovic](https://github.com/aleksandar-todorovic)
 
+### Articles
```
Comment (aleksandar-todorovic) [inline README.md]: Can you please change the heading to "Bash Learning Resources"?

"Articles" kind of makes it sound like there would be articles on other subjects.

**Comment by aleksandar-todorovic** [comment]
I'm satisfied with the content in there. Just change the heading and I'll accept the pull request. Thanks for contributing!

**Comment by aleksandar-todorovic** [comment]
Oh, forgot to mention: you've added a book from the WikiBooks, so I'm not gonna require a personal comment to add it. If it were from any other source, I would.

**Comment by gutierri** [comment]
@aleksandar-todorovic  Ready!

**Pull request merged**